webpack.config.js

```diff
@@ -1,117 +1,109 @@
 var _ = require('lodash')
 
 var webpack = require('webpack')
-var CommonsChunkPlugin = require('webpack/lib/optimize/CommonsChunkPlugin')
+//var CommonsChunkPlugin = require('webpack/lib/optimize/CommonsChunkPlugin')
 var ProgressPlugin = require('webpack/lib/ProgressPlugin')
 
 var pathutil = require('./lib/util/pathutil')
+var path = require('path')
 var log = require('./lib/util/logger').createLogger('webpack:config')
 
 module.exports = {
-  webpack: {
-    context: __dirname
-    , cache: true
-    , entry: {
-      app: pathutil.resource('app/app.js')
-      , authldap: pathutil.resource('auth/ldap/scripts/entry.js')
-      , authmock: pathutil.resource('auth/mock/scripts/entry.js')
+  mode: 'production',
+  context: __dirname,
+  cache: true,
+  entry: {
+    app: pathutil.resource('app/app.js'),
+    authldap: pathutil.resource('auth/ldap/scripts/entry.js'),
+    authmock: pathutil.resource('auth/mock/scripts/entry.js')
+  },
+  output: {
+    //path: pathutil.resource('build'),
+    path: '/tmp/build/res/build',
+    publicPath: '/static/app/build/',
+    filename: 'entry/[name].entry.js',
+    chunkFilename: '[id].[hash].chunk.js'
+  },
+  stats: {
+    colors: true
+  },
+  
+  resolve: {
+    modules: [
+      "node_modules", 
+      //path.resolve(__dirname, "bower_components"),  // works
+      '/tmp/build/bower_components',
+      //path.resolve(__dirname, "res/web_modules" ), // works
+      '/tmp/build/res/web_modules',
+      //path.resolve(__dirname, "res/app/components" )// works
+      '/tmp/build/res/app/components'
```

Comment:
I'd prefer something without absolute hardcoded path. Something like:
```
modules: [
      'node_modules',
      'bower_components',
      'web_modules',
      path.resolve(__dirname, "res/app/components" )
]
```

Reply:
I agree that a relative path is needed. This is part of what is breaking `stf local` also I believe.

I spent far too long fighting with the pathing and order of precedence here and once I got it working I threw in the towel and stopped changing it.

As mentioned ultimately bower modules should go away entirely anyway at some point...